website/content/api-docs/index.mdx

```diff
@@ -123,9 +126,9 @@ endpoints that consume parameters use `POST` instead and put the parameters in
 the request body.
 
 You can list secrets as well. To do this, either issue a GET with the query
-parameter `list=true`, or you can use the `LIST` HTTP verb. For the `kv`
-backend, listing is allowed on directories only, and returns the keys in the
-given directory:
+parameter `list=true`, or you use the `LIST` HTTP verb. For the `kv` secrets 
```

Comment:
You may want to put caveat portion of this statement in italics.

Reply:
@benashz do you mean the `or` or which portion do you think should be made italic?

Reply:
I think we can ignore this comment. I may have misinterpreted it.